graphene_django/filter/fields.py

```diff
@@ -68,7 +87,7 @@ def filter_kwargs():
                 if k in filtering_args:
                     if k == "order_by" and v is not None:
                         v = to_snake_case(v)
-                    kwargs[k] = v
+                    kwargs[k] = convert_enum(v)
```

Comment:
Does this mean `order_by` arguments can be an enum?

Reply:
no `order_by` have to be strings since it is supposed to be the name of the field you want to order by (and `to_snake_case` only works for strings)
`convert_enum` method  only transform `Enum`s and left other things untouched (so a string wouldn't get modified by the function)